pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
+    """
+    create a new policy from a JSON-file
+    """
+    import json
+
+    f = open(filename, 'r')
+    contents = f.read()
+    f.close()
+    params = json.loads(contents)
```

Comment:
I think the question here is: do we want the users to quickly get error-free json format from browser  and throw that into a file or we want them to write python-formatted files from scratch (then `ast.literal_eval()`  can be used)? 

(I'll have to first convert that file to something `ast` can understand so I can proceed with testing o.O)